roles/common/defaults/main.yml

```diff
@@ -95,6 +95,7 @@ common:
         - cyrus-sasl-devel
   python:
     base_venv: "/opt/openstack/base"
+    sensu_venv: "/opt/openstack/sensu"
```

Comment:
this path might confuse things...    can we add to `/opt/sensu/python-virtualenv` or something more specific for sensu ?